Readme.markdown

```diff
@@ -38,6 +39,7 @@ fallback\_icon\_url   | *HTTP image URL*         | If provided, a redirect to th
 |<https://icons.better-idea.org/icon?url=yelp.com&size=64>|![Icon for yelp.com](https://icons.better-idea.org/icon?url=yelp.com&size=64)|
 |<https://icons.better-idea.org/icon?url=yelp.com>|size missing|
 |<https://icons.better-idea.org/icon?url=httpbin.org/status/404&size=64>|![Icon for non-existent page](https://icons.better-idea.org/icon?url=httpbin.org/status/404&size=64)|
+|<https://icons.better-idea.org/icon?url=httpbin.org/status/404&size=64&fallback_icon_color=ff0000>|![Icon for non-existent page](https://icons.better-idea.org/icon?url=httpbin.org/status/404&size=64&fallback_icon_color=ff0000)|
```

Comment:
Why not use a nice, more colorful background color as example which is different from the default to make it even more obvious what this param is for?